src/main/java/edu/kis/powp/jobs2d/features/DriverChangeObserver.java

```diff
@@ -0,0 +1,14 @@
+package edu.kis.powp.jobs2d.features;
+
+import edu.kis.powp.observer.Subscriber;
+
+public class DriverChangeObserver implements Subscriber {
```

Comment:
The name should also indicate the behavior of this particular DriverChangeObserver (what does update() do).\
The convention of project use tabs (no spaces)